lib/src/components/NxList/__tests__/NxListButtonItem.test.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxListButtonItemProps } from '../types';
+import NxList from '../NxList';
+import { getMountedComponent, getShallowComponent } from '../../../__testutils__/enzymeUtils';
+
+describe('NxListButtonItem', function() {
+
+  const minimalProps: NxListButtonItemProps = {};
+  const getShallow = getShallowComponent(NxList.ButtonItem, minimalProps);
+  const getMounted = getMountedComponent(NxList.ButtonItem, minimalProps);
+
+  it('renders a correctly structured element - a button within an li', function() {
+    const children = [
+      <NxList.Text key="1">Test Item 1 Text</NxList.Text>
```

Comment:
Are the `children` props actually necessary in most of these tests?

Reply:
> Are the `children` props actually necessary in most of these tests?

Yes, because it is defined in `types.ts ` that children are required for `ButtonItem` and `LinkItem`

Reply:
> Yes, because it is defined in types.ts that children are required for ButtonItem and LinkItem

That doesn't look like the case to me,  both of those interfaces define `children` as an optional member (using `?`) and also define its type as `ReactNode` which has both `null` and `undefined` as subtypes.

Reply:
> > Yes, because it is defined in types.ts that children are required for ButtonItem and LinkItem
> 
> That doesn't look like the case to me, both of those interfaces define `children` as an optional member (using `?`) and also define its type as `ReactNode` which has both `null` and `undefined` as subtypes.

The children props have been removed.
1776aaebcc703414808783c567c01b0a222ce75f